cli/Makefile

```diff
@@ -11,20 +11,27 @@ BUCKET=gs://cft-cli
 INT_TEST_DIR=./bpmetadata/int-test
 SCHEMA_DIR=./bpmetadata/schema
 PROTO_DIR=./bpmetadata/proto
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.14
```

Comment:
We should probably add this to `regexManagers` in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/.github/renovate.json#L48

```
  "regexManagers": [
    {
      "fileMatch": ["(^|/)Makefile$"],
      "matchStrings": ["DOCKER_TAG_VERSION_DEVELOPER_TOOLS := (?<currentValue>.*?)\\n"],
      "datasourceTemplate": "docker",
      "registryUrlTemplate": "https://gcr.io/cloud-foundation-cicd",
      "depNameTemplate": "cft/developer-tools"
    }
  ]
```

Reply:
Done.